CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 ## [Unreleased] - 2019-04-26
 ### Added
 - Show version number in About dialog ([#28](https://github.com/cbrnr/mnelab/pull/28) by [Clemens Brunner](https://github.com/cbrnr))
+- Add Apply ICA to Tools menu that allows user to apply fitted ica to the current dataset by removing selected components from the signal. ([#43](https://github.com/cbrnr/mnelab/pull/43) by [Victor Férat](https://github.com/vferat))
```

Comment:
you should ad the change to a new version (not past unreleased), mnelab is already after `0.1.0`

Reply:
That's OK, unreleased means "currently unreleased", so this is going to be the next release.

Reply:
Minor nitpick: please rephrase to:
```
- Add "Apply ICA" to Tools menu that allows users to apply a fitted ICA solution to the current dataset ([#43](https://github.com/cbrnr/mnelab/pull/43) by [Victor Férat](https://github.com/vferat))

Reply:
Ah, sorry, I read it wrong.

Reply:
the dates are just a bit confusing, as this unreleased section has earlier date than the `0.1.0` release so I read it as past unreleased changes.

Reply:
Yes, I've changed the date to XX.XX.XXXX to avoid confusion.